hathor/event/base_event.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Dict
+
+
+@dataclass
+class BaseEvent:
+    def __init__(self, id: int, peer_id: str, timestamp: int, type: str, data: Dict):
```

Comment:
We do we need this `__init__` method instead of using what the `dataclass` would provide us?

Reply:
Oh, you're right, I didn't notice this.

Reply:
Fixed.